packages/core/usage-data/core-usage-data-base-server-internal/README.md

```diff
@@ -0,0 +1,6 @@
+# @kbn/core-usage-data-base-server-internal
+
+This package contains internal base types and constants for Core's server-side `usage-data` domain that are used
+across some other domains.
+
+The purpose of the package is mostly to break the cyclic dependency between the `core-usage-data` and `savedObjects` services.
```

Comment:
aside: Are we going to standardize on using the `*-base-*` package name tag for extra packages we need to break cyclic dependencies?
If so, can we generally assume that these packages won’t have associated public types packages?

At the end, in the audit and clean up phase, we should consider adding another folder for `shared` service dependencies and move the packages to those. WDYT?

Reply:
Good question. Regarding standardization, tbh I'm even starting to think that the current structure / suffix/prefix naming may not be the best one. But let's finish the migration the way the started it, and we then should discuss a bit regarding what we think we may want to improve.